game_server.js

```diff
@@ -20,9 +20,9 @@ linkHelper();
 let toExecute;
 //toExecute = new SplashScreen();
 //toExecute = new Draw();
-//toExecute = new LayerTest();
+toExecute = new LayerTest();
 //toExecute = new MoveTest();
-toExecute = new AllTest();
+//toExecute = new AllTest();
```

Comment:
A new task might be adding a game selector screen. This method of running a game isn't ideal. @YazeedLoonat we should talk about what that would look like